proxima-chapter-1.md

```diff
@@ -228,92 +230,107 @@ Then, let's proceed to prepare the environment for subsequent steps like:
 - Install the OCI Command Line Interface with `oci setup config`. That will be handy to access OCI resources from the Matrix command line terminal.
 - Setup **venv** with `python3.6 -m py36env`. Yes, we'll use some Python here.
 - Insert source py36env/bin/activate in a test script and wherever you feel appropriate
-- Install the Extra Packages for Enterprise Linux [https://fedoraproject.org/wiki/EPEL?source=:ex:of:::::RC_WWMK210721P00049:Medium])(EPEL)):   `sudo yum install -y oracle-epel-release-el7`
+- Install the [Extra Packages for Enterprise Linux (EPEL)](https://fedoraproject.org/wiki/EPEL?source=:ex:of:::::RC_WWMK210721P00049:Medium)): `sudo yum install -y oracle-epel-release-el7`
 
 #### Install, configure & test the MQTT Broker**
 
 As said, we'll use Mosquitto as the MQTT broker in OCI with sudo yum install mosquitto, and add some configuration, like
 
-- Create a password file (not really effective for the overall security posture, but I'd add credentials anyway)`: mosquitto_passwd -c <YourUsername> <YourPassword>.`
-- Move the file to the right directory: `sudo mv passwordfile /etc/mosquitto`.
-- Edit the default Mosquitto configuration found in /etc/mosquitto/mosquitto.conf in order to:
-- Identify and change the default port. If the service is likely to be exposed to the wild internet, leaving the default settings to 1883 would be a no-no, maybe, and add username & password previously set in the appropriate sections.
-- Restart Mosquitto: `sudo systemcl restart mosquitto`.
-    Test mosquitto with mosquitto clients, such as **mosquitto_sub** and **mosquitto_pub**
-    in a Matrix Terminal session, subscribe to all topics with mosquitto_sub -h 127.0.0.1 -p -t '#' -u <YourUsername> - <YourPassword>.
-- in another Matrix Terminal session, publish a message with:
-`mosquitto_pub -h 127.0.0.1 -p <port>-t 'Test' -u <YourUsername> -P <YourPassword> -m '{"Name":"Jack Torrance"}'.`
-- if all is well, you should see the message on the subscriber's terminal session: in such a case, grab a coffee or a nice cup of tea, as we're going secure our MQTT Server first, and then installling the OCI SDK and examples for Python.
+1. Create a password file (not really effective for the overall security posture, but I'd add credentials anyway) `: mosquitto_passwd -c <YourUsername> <YourPassword>.`
+2. Move the file to the right directory: `sudo mv passwordfile /etc/mosquitto`.
+3. Edit the default Mosquitto configuration found in /etc/mosquitto/mosquitto.conf in order to identify and change the default port. 
+    
+    If the service is likely to be exposed to the wild internet, leaving the default settings to 1883 would be a no-no, maybe, and add username & password previously set in the appropriate sections.
+4. Restart Mosquitto: `sudo systemcl restart mosquitto`.
+    
+    Test mosquitto with mosquitto clients, such as `mosquitto_sub` and `mosquitto_pub`.
+     
```

Comment:
You can include paragraphs in a list by adding a blank line and indenting 4 spaces/1 tab beyond the level of the parent item. Multiple paragraphs should have blank lines between them.